iris/static/js/lecturer_session.js

```diff
@@ -1,37 +1,41 @@
 const button_start = document.getElementById("button_start");
 const button_stop = document.getElementById("button_stop");
-const feedbackmsg = document.getElementById("lecturer_message")
+const feedbackmsg = document.getElementById("lecturer_message");
 let socket = io();
 socket.emit('join', {'course_id': courseID});
 
-if(sessionActive == "True"){
+function DisableStart(){
```

Comment:
Hmm, maybe? I tend to use upper case to better distinguish them from normal variables.